### PR TITLE
[keyboard] add new keyboard zoom pad

### PR DESCRIPTION
**Pull request by spbgzh**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x]  I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
@spbgzh 
Keymaps contain a Layer 1 but no means to access it. Could you rectify?

**Comment by spbgzh** [comment]
@dunk2k fixed

**Review by dunk2k — APPROVED** [review]

File: keyboards/meletrix/zoom_pad/keymaps/southpaw/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
```
Comment (zvecr) [inline keyboards/meletrix/zoom_pad/keymaps/southpaw/rules.mk:1]: Enabling here via makes this keymap somewhat pointless. A user would just flash the `via` keymap and configure the layout options within its UI.

File: keyboards/meletrix/zoom_pad/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Flashing example for this keyboard:
+
+    make meletrix/zoom_pad:default:flash
+
+To reset the board into bootloader mode, do one of the following:
+
+* Tap the Reset switch mounted on the PCB
+* Hold the Esc key while connecting the USB cable (also erases persistent settings)
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (zvecr) [inline keyboards/meletrix/zoom_pad/readme.md:23]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (the top left key) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the PCB
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by zvecr** [comment]
If you are going to request a review, you should probably not ignore both the suggested changes.

**Comment by zvecr** [comment]
> If you are going to request a review, you should probably not ignore **both** the suggested changes.

File: keyboards/meletrix/zoom_pad/info.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B4", "pin_b": "D3", "resolution": 2}
+        ]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (drashna) [inline keyboards/meletrix/zoom_pad/info.json:32]: This should be reformatted to match the numpad_6x4 layout:
https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_6x4

And should be named as such.

https://docs.qmk.fm/#/feature_layouts

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by spbgzh** [comment]
@drashna I made change

**Pull request closed without merging**